app/models/setting.py

```diff
@@ -30,6 +30,8 @@ class Setting(db.Model):
     secret = db.Column(db.String)
     # Static domain
     static_domain = db.Column(db.String)
+    # Order Expiry Time
+    order_expiry_time = db.Column(db.Integer, default=15)
```

Comment:
Write # minutes in front of it

Reply:
@kushthedude I guess you missed to put a comment over here.

Reply:
@shreyanshdwivedi I have specified the `minutes` above the field, Writing `minutes` in front of field definition, Would fail hound